app/mailers/user_mailer.rb

```diff
@@ -4,6 +4,11 @@ def welcome(user_id, subject)
     mail(to: @user.email, subject: subject)
   end
 
+  def welcome_shop_owner(shop_owner_id, subject)
```

Comment:
On a second thought, I feel think is rework here. If shop_owner was passed directly, we don't need to find it using shop_owner_id anymore. So we have a one line method. What do you think?